src/components/Discord.tsx

```diff
@@ -7,35 +7,27 @@ const Discord = () => {
 
   return (
     <div
-      className="center justify-center w-full py-16 scroll-mt-[136px] flex gap-y-8"
+      className="center justify-center w-full py-16 scroll-mt-[136px] flex gap-y-8 gap-x-6"
       id="join-us"
     >
-      <div className="border border-blue rounded mr-4">
-        <label className="text-2xl font-bold text-center text-yellow-400">
-          No Pressure{" "}
-        </label>
-        <label className="text-yellow-400">
-          started as a European counterpart to
-        </label>
-        <br></br>
-        <label>
-          <a className="text-yellow-200" href={wowMadeEasy}>
+      <div className="basis-2/4 w-px border border-blue rounded p-16 flex flex-col gap-y-8">
+        <p className="text-yellow-400">
+          <strong className="font-bold text-center text-yellow-400">
+            No Pressure
+          </strong>{" "}
+          started as a European counterpart to{" "}
+          <a className="text-light-200" href={wowMadeEasy}>
             WoW Made Easy
           </a>
-        </label>
-        <label className="text-yellow-400">
-          {" "}
-          - a community built on enjoying.
-        </label>
-        <p className="text-yellow-400">
-          Dragonflight together, without the pressure
+          - a community built on enjoying Dragonflight together, without no
+          pressure
         </p>
-        <div className="my-4 w-[500px] h-[185px] border-blue border rounded flex items-center justify-evenly">
+        <div className="my-4 p-8 border-blue border rounded-xl flex items-center justify-evenly">
           <img src={logo} />
```

Comment:
I think just a small right margin on the image (maybe `mr-2`/`mr-4`) would go nicely here